ports/litex/common-hal/digitalio/DigitalInOut.c

```diff
@@ -61,36 +61,55 @@ void common_hal_digitalio_digitalinout_deinit(digitalio_digitalinout_obj_t *self
 void common_hal_digitalio_digitalinout_switch_to_input(
         digitalio_digitalinout_obj_t *self, digitalio_pull_t pull) {
     (void)pull;
+#ifdef FOMU
```

Comment:
Should this be dependent on the board or can we depend on something generated with the SoC?

Reply:
Yes, I think we can be checking if register definitions in the `csr.h` exist instead, that makes sense.